### PR TITLE
Feature/limit items; choose languange and currency

### PR DESCRIPTION
**Pull request by gustavotr**
Fix issue #19 
Fix issue #24

### CODE REVIEW

**Comment by pocesar** [comment]
hey @gustavotr can you merge both PRs into one? they will conflict since some of the code is being changed in both PRs. or should only this one be prioritized?

**Comment by gustavotr** [comment]
Sure, I will fix this one and cancel the other one.

**Comment by gustavotr** [comment]
@pocesar done

File: INPUT_SCHEMA.json [hunk 1]
```diff
@@ -52,6 +52,18 @@
       "description": "If checked results includes reviews data",
       "default": false
     },
+    "maxReviews": {
+      "title": "Limit reviews for each place",
+      "type": "number",
```
Comment (pocesar) [inline INPUT_SCHEMA.json]: need to set the `editor` here and for `maxItems` to `number` (and the type being `integer`)

use `apify vis` to check if the schema is valid

File: INPUT_SCHEMA.json [hunk 2]
```diff
@@ -70,14 +82,526 @@
...
+      },
+      "default": {
+        "useApifyProxy": true
+      },
       "editor": "proxy"
     }
   },
-  "required": ["locationFullName", "includeReviews"]
-}
+  "required": [
```
Comment (pocesar) [inline INPUT_SCHEMA.json]: while you're at it, exchange this array to `["proxyConfiguration"]` only

File: src/main.js [hunk 3]
```diff
@@ -44,21 +44,29 @@ Apify.main(async () => {
...
         if (!proxyConfiguration.useApifyProxy) {
             throw new Error('Proxy required! Usage of Apify Proxy is required');
         }
     }
+
     global.USE_PROXY = proxyConfiguration.useApifyProxy;
     global.PROXY_GROUPS = proxyConfiguration.apifyProxyGroups;
 
     global.LANGUAGE = input.language || 'en_USA';
+    const currency = input.currency || 'CZK';
```
Comment (pocesar) [inline src/main.js]: default should be `'USD'`

File: INPUT_SCHEMA.json [hunk 4]
```diff
@@ -70,14 +82,526 @@
...
+        "Xhosa",
+        "Yiddish",
+        "Yoruba",
+        "Zhuang, Chuang"
+      ]
+    },
+    "currency": {
+      "title": "Currency",
+      "type": "string",
+      "editor": "select",
```
Comment (pocesar) [inline INPUT_SCHEMA.json:466]: place the `default` and `prefill` to `USD`

**Review by pocesar — CHANGES_REQUESTED** [review]
nitpick: I can't test on the platform while the INPUT schema is invalid

**Comment by pocesar** [comment]
I'm going to merge this, but need to fix some issues, the user agent is giving away that it's using axios, as I can't run this on the platform properly

**Pull request merged**